inc/files.php

```diff
@@ -142,7 +142,7 @@ function ppom_create_thumb_for_meta( $file_name, $product_id, $cropped = false,
  * @return string The new file name.
  */
 function ppom_create_unique_file_name( $file_name, $file_ext ) {
-	return $file_name . "." . base64_encode( substr( wp_hash_password( $file_name ), 0, 8 ) ) . "." . $file_ext;
+	return $file_name . "." . base64_encode( substr( wp_hash_password( $file_name ), 0, 16 ) ) . "." . $file_ext;
```

Comment:
I think was part was way overkill. I think we can simplify to just:

```php
$file_name . "." . wp_hash( $file_name ) . "." . $file_ext;
```

Reply:
@Soare-Robert-Daniel, as you suggested, we added and tested with wp_hash; however, we got the same string every time for the same input string. Can you please test your end with the same function and let me know if changes are required.

Reply:
You are right. I forgot about the random part to plug into the hash. This one is passing the test.

```php
function ppom_create_unique_file_name( $file_name, $file_ext ) {
    $seed = $file_name . microtime( true ) . mt_rand();
    return $file_name . "." . wp_hash( $seed ) . "." . $file_ext;
}
```

Reply:
@Soare-Robert-Daniel I've implemented as per your suggestion with my latest commit, can you please check and let me know if changes are needed.